### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by richleland**

### CODE REVIEW

**Comment by aydrian** [comment]
LGTM If you don't have any issues with my changes, let's merge.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+## Submitting Issues
+
+* You can create an issue [here](https://github.com/sparkpost/python-sparkpost/issues/new), but
+  before doing that please read the notes below on debugging and submitting issues,
+  and include as many details as possible with your report.
+* Include the version of python-sparkpost you are using.
+* Perform a [cursory search](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aissue+user%3Asparkpost+repo%3Apython-sparkpost)
+  to see if a similar issue has already been submitted.
+
+## Guidelines
```
Comment (richleland) [inline CONTRIBUTING.md]: This seems like it should maybe go after Contribution Steps. Maybe as a ``###`` ?

Comment (aydrian) [inline CONTRIBUTING.md]: The guidelines section? I could agree with that. I moved it around a couple times. Couldn't decide.

Comment (richleland) [inline CONTRIBUTING.md]: Yep

**Comment by aydrian** [comment]
Updated. Take a look.

**Comment by aydrian** [comment]
Closes #29

**Pull request merged**